students/AurelPerianu/Lesson6/test_mailroom_4.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import unittest
+import mailroom_4 as mailroom
+import os
+
+
+class MailroomTest(unittest.TestCase):
+
+    def test_donor_amount(self):
+        self.assertEqual(mailroom.donors['Donor A'], [3580, 34124.31, 7654])
```

Comment:
This test is not necessary since it's really just testing the python dict works as expected